src/borrow_tracker/tree_borrows/mod.rs

```diff
@@ -407,8 +430,16 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                 self.ecx
             }
 
+            // Regardless of whether special behavior is enforced for `Unique`
+            // (`unique_is_unique`), we always treat `Box`es the same way, though they get an extra
+            // read access.
```

Comment:
What is the extra read access?

Reply:
My bad, this is outdated. It should say instead that `Unique` gets a `zero_size` while `Box` has an actual non-zero-sized reborrow.